### PR TITLE
docs(@clayui/navigation): Adds documentation for Breadcrumbs

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

File: next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [API](#api)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/alerts/">Alerts Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
```
Comment (bryceosterhaus) [inline next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx]: `<a href="https://liferay.design/lexicon/core-components/alerts/">Alerts Pattern</a>`

Is this correct?

File: next.clayui.com/src/components/clay/Breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import React from 'react';
+import {ClayBreadcrumbNav} from '@clayui/navigation';
+
+const spritemap = '/images/icons/icons.svg';
+
+const BreadcrumbCode = `const Component = () => {
+	return (
+		<ClayBreadcrumbNav
+			ellipsisBuffer={3}
+			items={[
```
Comment (bryceosterhaus) [inline next.clayui.com/src/components/clay/Breadcrumb.js]: for the sake of bloat, lets maybe only do 6 or so items.

Comment (bryceosterhaus) [reply next.clayui.com/src/components/clay/Breadcrumb.js]: or create some sort of map to condense this code into a smaller chunk

File: next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx (hunk 1 above)
Comment (diegonvs) [reply next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx]: nope 😂

File: next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/navigation/breadcrumb/">Breadcrumb Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<Breadcrumbs />
+
+We recommend that you review the use cases in the <a href="https://storybook-clayui.netlify.com/?path=/story/components-claynavigation--w-breadcrumb">Storybook</a>.
```
Comment (matuzalemsteles) [inline next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx]:
```suggestion
We recommend that you review the use cases in the <a href="https://storybook.clayui.com/?path=/story/components-claynavigation--w-breadcrumb">Storybook</a>.
```

**Comment by diegonvs** [comment]
Reopened at #2506

**Pull request closed without merging**